module/VuFind/src/VuFind/Log/Handler/PostHandler.php

```diff
@@ -27,43 +27,31 @@
  * @link     https://vufind.org Main Site
  */
 
-namespace VuFind\Log\Writer;
+namespace VuFind\Log\Handler;
 
 use Laminas\Http\Client;
+use Monolog\Handler\AbstractProcessingHandler;
+use Monolog\LogRecord;
 
 use function is_array;
 
 /**
- * This class extends the Laminas Logging to sent POST messages over HTTP
+ * This class extends the Monolog Logging to sent POST messages over HTTP
  *
  * @category VuFind
  * @package  Error_Logging
- * @author   Chris Hallberg <challber@villanova.edu>
+ * @author   Sambhav Pokharel <sambhav.pokharel@gmail.com>
```

Comment:
Same thing here. (And for the record, it does annoy me that we can't have the class simply inherit the annotations from the file by default; that would be a lot easier and save a lot of bytes on disk... but PHPDoc and/or our style checker insists on doing it this way).